### ISSUE TITLE
Round trip metadata

### ISSUE THREAD

**Issue by Neosettler** [comfyanonymous/ComfyUI#3739]
Hi, I've been pulling my hair out on this one and decided to seek for help.

After sending a "queue prompt" command line, I'm listening to the sever responses. One of them is "executed":

"data": {
        "node": "53",
        "output": {
            "gifs": [
                {
                    "filename": "00001.mp4",
                    "format": "video/h264-mp4",
                    "subfolder": "",
                    "type": "temp"
                }
            ]
        },
        "prompt_id": "e4e3e945-00c4-487d-ad90-3dbe9b02e99d"
    },
    "type": "executed"
    
    I'm wondering how could we get extra metadata in there?
    
     for instance under "output":    
    "extra_metadata": "myMeatadata",
    
    So basically, extra "metadata" has to be passed to the workflow (encapsulated in a node or else) and sent back whenever the output is ready.

Thank you,
